llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1341,7 +1341,102 @@ void SpirvLowerGlobal::addCallInstForOutputExport(Value *outputValue, Constant *
     m_builder->CreateWriteGenericOutput(outputValue, location, locOffset, elemIdx, maxLocOffset, outputInfo, vertexIdx);
   }
 }
+Value *SpirvLowerGlobal::getConstantInt32(unsigned c) {
+  return ConstantInt::get(Type::getInt32Ty(*m_context), c);
+}
+// =====================================================================================================================
+// Inserts instructions to load needed members from input/ouput location.
+//
+// @param inOutTy : Type of this input/output member
+// @param addrSpace : Address space
+// @param indexOperands : Index operands
+// @param operandIdx : Index of the index operand in processing
+// @param inOutMetaVal : Metadata of this input/output member
+// @param locOffset : Relative location offset of this input/output member
+// @param interpLoc : Interpolation location, valid for fragment shader (use "InterpLocUnknown" as don't-care value)
+// @param auxInterpValue : Auxiliary value of interpolation (valid for fragment shader):
+//                       - Sample ID for "InterpLocSample"
+//                       - Offset from the center of the pixel for "InterpLocCenter"
+//                       - Vertex no. (0 ~ 2) for "InterpLocCustom"
+// @param insertPos : Where to insert instructions
+Value *SpirvLowerGlobal::loadNeededInOutMembers(Type *inOutTy, unsigned addrSpace,
+                                                const std::vector<Value *> &indexOperands, unsigned operandIdx,
+                                                Constant *inOutMetaVal, Value *locOffset, unsigned interpLoc,
+                                                Value *auxInterpValue, Instruction *insertPos) {
+  assert(m_shaderStage == ShaderStageFragment);
```

Comment:
Make comments to explain why this function is just for fragment shader. You can say:
// This function is currently just used for fragment shader to ...

Reply:
good idea.